README.rst

```diff
@@ -106,23 +106,38 @@ Using the utilities in OpenWISP modules
 Using the ``admin_theme``
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-For using the customized admin theme,
+For using the customized admin theme make sure you've added
+``openwisp_utils.admin_theme`` to ``INSTALLED_APPS`` in ``settings.py``
 
 * Make sure you've added ``openwisp_utils.admin_theme`` to ``INSTALLED_APPS``
   in ``settings.py``
+* Import ``admin_site`` and make changes like following into your ``urls.py`` file which contains ``admin`` urls.
+
+.. code-block:: python
+
+    from django.conf.urls import url
+    from openwisp_utils.admin_theme.site import admin_site
+
+    urlpatterns = [
+        # other url patterns
+        url(r'^admin/', admin_site.urls),
+    ]
+
+Alternatively you can use ``openwisp_admin()`` function, but its usage is discouraged.
+
 * Add the following into your ``urls.py`` file which contains ``admin`` urls.
 
 .. code-block:: python
 
-    from django.conf.urls import include, url
+    from django.conf.urls import url
 
     from openwisp_utils.admin_theme.admin import admin, openwisp_admin
 
     openwisp_admin()
```

Comment:
remove this

Reply:
@nemesisdesign I have to remove whole section about `openwisp_admin` from README, right?

Reply:
what section?

Reply:
@nemesisdesign I already made changes in `README` is it now like supposed to be?